### PR TITLE
Fixes#882 Adds validations in session-speaker route

### LINKED ISSUE #882: Fix validations in session-speaker-form component
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Fix the validations for dropdowns in session-speaker-form component.

### PR DESCRIPTION
**Pull request by ParthS007**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Added validations in session-speaker route

Partially Fixes #882

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-1057-open-event-frontend-fossasia.surge.sh

**Comment by ParthS007** [comment]
@geekyd @CosmicCoder96 Please review and suggest changes if any.Thanks

**Comment by ParthS007** [comment]
@geekyd @CosmicCoder96 Please review and suggest changes if any.Thanks

File: app/components/forms/wizard/sessions-speakers-step.js [hunk 1]
```diff
@@ -41,6 +41,33 @@ export default Component.extend(EventWizardMixin, FormMixin, {
...
+          identifier : 'announcement',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter an Announcement')
+            }
+          ]
+        },
+        privateLink: {
+          identifier : 'privatelink',
```
Comment (CosmicCoder96) [inline app/components/forms/wizard/sessions-speakers-step.js]: make this `private_link`

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 2]
```diff
@@ -155,7 +155,7 @@
     <div class="field">
       <label>{{t 'Private Link'}}</label>
       <div class="ui action input fluid">
-        <input type="text" value="{{concat data.event.url '/cfs/' data.speakersCall.hash}}" placeholder="{{concat data.event '/cfs/' data.speakersCall.hash}}" readonly>
+        <input type="text" name="privatelink" value="{{concat data.event.url '/cfs/' data.speakersCall.hash}}" placeholder="{{concat data.event '/cfs/' data.speakersCall.hash}}" readonly>
```
Comment (CosmicCoder96) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs]: same here

**Comment by niranjan94** [comment]
Deployment link: https://pr-1057-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-1057-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-1057-open-event-frontend-fossasia.surge.sh

**Comment by ParthS007** [comment]
@geekyd waiting for your review :smiley:

**Comment by geekyd** [comment]
@ParthS007 The PR partially fixes the issue, the issue is in the speaker-session form(https://github.com/fossasia/open-event-frontend/blob/development/app/components/forms/session-speaker-form.js), we need to add validations to the dropdowns. 

Merging this. Thanks :)

**Pull request merged**